contrib/roles/linux/validation/vars/ubuntu.yml

```diff
@@ -0,0 +1,10 @@
+---
+master_service_names:
+  - "kube-apiserver"
+  - "kube-controller-manager"
+  - "kube-scheduler"
+  - "etcd3"
+  - "ovn-k8s-watcher"
+
+minion_service_names:
+  - "kubelet"
```

Comment:
You should also add the ``ovnkube-gateway-helper `` service here. I would suggest renaming this to ``ovn-kubernetes-node`` to have consistency with the case when you don't have the gateway initialized.

Reply:
Good point!

However, there is something else. [This PR](https://github.com/openvswitch/ovn-kubernetes/pull/555/files#diff-5be481425bd1879c76ae3f6b804381c1R125) adds a new systemd service simply called `ovnkube`.

Basically, on Windows, whether gateway is enabled or not, the service is named `ovn-kubernetes-node`.

While on Linux, when gateway is enabled, we have `ovnkube-gateway-helper`. And when gateway is disabled, the service is called `ovnkube`.

I'd suggest to have the service simply called `ovnkube` on both Windows and Linux, regardless of the `init_gateway` flag. What do you think ?

Reply:
Let's go with ``ovn-kubernetes-node``, it's better than ``ovnkube``.